R/create-individual.R

```diff
@@ -0,0 +1,87 @@
+#' Creates an individual using the PKSim Database.
+#'
+#' @param path A string representation of a path, with path entries separated
+#'
+#' @return An array of \code{ParameterValue} containing the value of each individual parameter
+#' @examples
+#'
+#' @export
+createIndividual <- function(originData, useDistribution = FALSE, moleculeOntogenies = NULL) {
+  validateIsOfType(originData, OriginData)
+  validateIsLogical(useDistribution)
+  validateIsOfType(moleculeOntogenies, MoleculeOntogeny, nullAllowed = TRUE)
+  individualFactory <- rClr::clrCallStatic("PKSim.R.Api", "GetIndividualFactory")
+  moleculeOntogenies <- c(moleculeOntogenies)
+
+  individualCharacteristics <-  rClr::clrNew("PKSim.R.Domain.IndividualCharacteristics")

```

Comment:
I have defined in PKSim a NEW object IndividualCharacteristics that is basically the argument used to the function.
Why? because I spent two hours trying to pass array of object from R to .NET and it did not work. So I am creating one object that has an array as member and all is good